deploying-daml-pathway.json

```diff
@@ -0,0 +1,13 @@
+{
+    "title": "Deploying with DAML",
+    "private": false,
+    "courses": [
+      {
+        "external_link": "TODO",
+        "title": "Deploy DAML on Fabric",
+        "description": "Get familiar with running an application on the Fabric distributed ledger",
+        "course_id": "daml-on-fabric"
```

Comment:
I guess the course will be deploying DAML apps on different ledgers. And then Fabric is just one scenario with the course